requirements/base.txt

```diff
@@ -24,4 +24,4 @@ six==1.11.0               # via docker, docker-compose, docker-pycreds, dockerpt
 texttable==0.9.1          # via docker-compose
 urllib3==1.22             # via requests
 websocket-client==0.47.0  # via docker, docker-compose
-pygments==2.6.1           # Need this pacakge to render readme.rst
+pygments==2.3.1           # Need this pacakge to render readme.rst
```

Comment:
@emzubair why are we changing the requirements file?

Reply:
Version `2.6.1` is not compatible with python 2.7.

Reply:
It's recommended to install python 3.6.5